### ISSUE TITLE
v39 planning

### ISSUE THREAD

**Issue by HonkingGoose** [renovatebot/renovate#25608]
### Describe the proposed change(s).

- [ ] https://github.com/renovatebot/renovate/pull/32095
- [x] https://github.com/renovatebot/renovate/issues/32068
- [x] https://github.com/renovatebot/renovate/issues/32136

#### Copied from "old" v38 deferred list

Source: _Deferred to future major_ list from issue #27992.

- [ ] #6842
- [ ] #16359 - blocked due to JSON5 migration
- [ ] #21442 - blocked by renovateCompatibility (#19980)
- [ ] #21446

### Related issues

- #27992
- #23911
- #21134

**Comment by HonkingGoose** [comment]
@rarkins can you create a new `v39` label? Then I can label this issue, and the todos listed in it as to do for `v39`. 😄

**Comment by viceice** [comment]
Closing, v39 released
